panel/widgets/icon.py

```diff
@@ -42,3 +42,22 @@ def _update_icon(self):
         icon_is_svg = self.icon.startswith('<svg')
         if icon_is_svg and not self.active_icon:
             raise ValueError('The active_icon parameter must not be empty if icon is an SVG.')
+
+
+class ToggleIcon(ClickableIcon):
+
+    value = param.Boolean(default=False, doc="""
+        Whether the icon is toggled on or off.""")
+
+    _widget_type = _PnToggleIcon
+
+
+class ButtonIcon(ClickableIcon):
+
+    clicks = param.Integer(default=0, doc="""
+        The number of times the button has been clicked.""")
+
+    active_duration = param.Integer(default=75, doc="""
```

Comment:
It also controls how long the button is disabled right?

Reply:
Yup, added mention. I'm also torn between naming it `disabled_duration` vs `active_duration` since `active_icon` and `disabled` both get affected.

Reply:
Maybe just toggle_duration.